logistics_planning_base/views/stock_picking_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="action_stock_picking_logistics_schedule" model="ir.actions.act_window">
+        <field name="name">Logistic Schedule</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">logistics.schedule</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[("picking_id", "=", active_id)]</field>
+    </record>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form.inherit</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
```

Comment:
Instead of using groups for every field, use `groups_id`. I know it will be removed in v16, but...